tmt-app/src/Components/ResultCards.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Fragment } from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Container,
+} from "@mui/material";
+import { ReactComponent as EmotionalIntelligence } from "../Images/icons/emotional_intelligence.svg";
+import { Link } from "react-router-dom";
+import TopicIcon from "../constants/SpriteSheet.svg";
+
+function ResultCards({ cardData }) {
+  const IconSvg = ({ topic, size }) => (
+    <svg fill="white" width={size} height={size}>
+      <use href={`${TopicIcon}#icon-${topic}`} />
+    </svg>
+  );
```

Comment:
Nice! lets make this much more generic though. We want to consider the reuse of this elsewhere in the application for ANY icons that will be added in later. 

Couple things, the file could be just `sprite.svg` or `spriteSheet.svg` is fine too. As a nit pick and potentially functional tip, pay attention to your capitalizations of your files, variables, routes, etc. In some contexts it matters how you format them. 

I think a simple rule to follow at least for react development. PascalCasing is for components, camelCasing for variables, SNAKE_CAPS for constants, kebab-case for file naming and route paths. It can vary per project and can also be preference, but i think thats a pretty safe assumption to be had when looking at other projects too. cc: @michachan for thoughts